test_cases/autocomplete_endonyms.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "autocomplete endonyms",
+  "notes": "https://github.com/pelias/wof-admin-lookup/pull/314",
+  "endpoint": "autocomplete",
+  "priorityThresh": 1,
+  "tests": [
+    {
+      "id": "1.0",
+      "status": "pass",
+      "user": "missinglink",
+      "description": [
+        "Data default label is English"
+      ],
+      "in": {
+        "text": "Reichstag Germany"
+      },
+      "expected": {
+        "properties": [
+          {
+            "label": "Reichstag, Berlin, Germany"
```

Comment:
We generally have a policy of not testing the `label` field for tests that aren't directly related to labels. Otherwise they lead to a lot of churn in the tests if/when labels change.

Maybe instead we test the name, locality, and country fields?